### ISSUE TITLE
Logo image optimization

### ISSUE THREAD

**Issue by jessicareed** [bppwrite/bppwrite.github.io#29]
I cannot figure out the rhyme or reason of logo display - there are no optimization suggestions in the Contributing document and it doesn't appear that resizing makes any difference. I have a Mother Bears logo that is 2185 x 2252 pixels, 300 resolution but it's displaying tiny. I have an IUCU logo image that is 900 x 133 pixels, 72 res and it's display is gigantic. Do you have any suggestions for logo image optimizations? Thank you!

**Comment by marcguyer** [comment]
Will you take a look at this @mattbatman? A I recall, this is by design due the disparate aspect ratios of the logos. Can you confirm. Some brief optimization guidelines for logos would probably be useful long-term.

**Comment by mattbatman** [comment]
Right now, all the logos are set to render as the same height on the website. Their width varies to keep the perspective ratio of the original image. This seemed to be the easiest way to let them flow and adjust to the width of the screen without creating big gaps of white space. We could come up with a different strategy.

**Comment by JSurigao** [comment]
Has there been any progress on coming up with a different strategy for this? There needs to be a way to give season sponsors roughly the same visual weight. Also, we do have a lead sponsor this year - Cook Group - but when I set it as lead, the rest of the logos get super wonky. 

![screen shot 2018-07-09 at 10 47 09 am](https://user-images.githubusercontent.com/33761733/42458924-d110c5be-8368-11e8-9237-f131ad2355dd.png)

**Comment by mattbatman** [comment]
> Also, we do have a lead sponsor this year - Cook Group - but when I set it as lead, the rest of the logos get super wonky.

Are you using the `2019` branch? I can fix this and push the changes there.

> Has there been any progress on coming up with a different strategy for this? There needs to be a way to give season sponsors roughly the same visual weight.

No, not really. In lieu of requesting logos from each sponsor of similar dimensions, I feel like the best strategy here will be to categorize each logo based on their dimensions, fit them on the same line, and try to make each logo on the same line roughly the same size (and try to balance each line).

I tried and failed to do this well earlier in the year, but I can try it again along with fixing the lead sponsor bug. We could work from there.

**Comment by JSurigao** [comment]
Yes, the 2019 branch. I currently have "lead" set to "false" for Cook Group there though. 

If you feel you can take another stab at arranging them individually, please do. I wish I could help brainstorm more, but I really don't know what our options are with this kind of site. Keep me posted and let me know if/how I can help. 

I'd love to merge this branch live as soon as possible, since we've already announced the new season. If this could take awhile, I may want to go ahead and merge it in a "good enough for now" state, then work on these changes in another branch. Let me know what you think. 

Thanks.

**Comment by mattbatman** [comment]
I think I can improve it from where it's at now before tomorrow.

**Comment by JSurigao** [comment]
Great, thanks!

**Comment by mattbatman** [comment]
@JSurigao Go ahead and pull the latest changes to the `2019` page and let me know why you think of the layout of the sponsors.

**Comment by JSurigao** [comment]
Ok. This is the first time I've done this - do I just do a "pull request"? I just want to make sure I don't do the wrong thing and end up merging everything. :)

**Comment by JSurigao** [comment]
Please disregard my last questions. I found the "pull" button.

**Comment by JSurigao** [comment]
Looks much better, thanks! One issue - the bottom three seem larger than the rest. It's actually ok if the IAC and BCCF logos are set apart, because they are granting entities, not traditional business sponsors, but having Bloom appear larger than others makes it look as though they support us at a higher/special level, and they do not. If it helps, I have a logo from them with a longer, horizontal orientation like the others. You could put that in the 2nd row? Let me know. 

If it's just easier for you to make those bottom three a little smaller, that may be ok too. 

I only see these changes on the home page right now, but that may be on purpose.

**Comment by mattbatman** [comment]
I think your browser is just remembering the old styles on the other pages. I opened a ticket (#42) about this. There are ways to disable this in your browser, but you could also just try refreshing the page a few times. My guess is that this is what you're experiencing with #41, too.

A wider logo for Bloom would likely be better, in combination with making the other two smaller.

**Comment by JSurigao** [comment]
The logos are consistent on all pages now, after refreshing a few more times. 

Should I just replace the logo on my end or send it to you?

**Comment by mattbatman** [comment]
Go ahead and send it to batman.matt@gmail.com.

**Comment by mattbatman** [comment]
@JSurigao The logo size modifications should be done on the 2019 branch.

I had just asked about what URL you were looking at in the other ticket because, for whatever reason, I still haven't gotten the 2018-2019 season to be viewable. In the 2019 branch, I still see the 2017-2018 season. The route `/series/2018-2019/` hits a 404.

**Comment by JSurigao** [comment]
@mattbatman I can't get the 2018-2019 season viewable from my laptop on the 2019 branch either. The files are all updated and seem correct in the directory, but what I'm seeing in the browser is still the 17-18 shows. The new season announcement hero shows up, but not the shows. I'm home sick, so I guess I'll have to check when back in office where I know it works tomorrow. 

I had to clear my browser cache and start over to see the Bloom logo change, but I see it now. I think Bloom could actually be a tiny bit bigger for legibility, but it's fine as it is if you're sick of messing with it. They are only an in-kind sponsor and I know it's quite a bit wider than the others. 

Thanks!

**Comment by JSurigao** [comment]
I wonder if this has anything to do with not being able to see the new season. I don't really know what it means, just wondering if that's what it's suppose to say in the terminal. 

<img width="624" alt="screen shot 2018-07-11 at 2 39 35 pm" src="https://user-images.githubusercontent.com/33761733/42592704-51540118-8518-11e8-9b40-e4fa2a99c5ee.png">
